sandbox/src/Bitcoin.tsx

```diff
@@ -232,6 +232,90 @@ function BtcSignPSBT({ bb02 }: Props) {
   );
 }
 
+function BtcSignMessage({ bb02 }: Props) {
+  const [coin, setCoin] = useState<bitbox.BtcCoin>('btc');
```

Comment:
remove this state, sign msg only works for `btc` anyway.

Reply:
Alright, coin is hardcoded to `btc` now